docs/source/mayavi/data.rst

```diff
@@ -518,7 +518,7 @@ objects or data files for Mayavi and TVTK.
 
  * Presentation on TVTK and Mayavi2 for course at IIT Bombay
 
-   https://svn.enthought.com/enthought/attachment/wiki/MayaVi/tvtk_mayavi2.pdf
+   In source: ./docs/pdf/tvtk_mayavi2.pdf
```

Comment:
Can link directly to github: https://github.com/enthought/mayavi/raw/master/docs/pdf/tvtk_mayavi2.pdf